docker/Dockerfile

```diff
@@ -0,0 +1,157 @@
+FROM phusion/baseimage:0.9.18
+
+# ------------------------------------------------------------------------------
+# Install Dependencies
+# ------------------------------------------------------------------------------
+
+RUN apt-get update 
+RUN apt-get install -y make
+RUN apt-get install -y wget 
+RUN apt-get install -y unrar 
+RUN apt-get install -y 	autoconf 
+RUN apt-get install -y 	automake 
+RUN apt-get install -y 	libtool 
+RUN apt-get install -y 	gcc 
+RUN apt-get install -y 	g++ 
+RUN apt-get install -y 	gperf 
+RUN apt-get install -y 	flex 
+RUN apt-get install -y 	bison 
+RUN apt-get install -y 	texinfo 
+RUN apt-get install -y 	gawk 
+RUN apt-get install -y 	ncurses-dev 
+RUN apt-get install -y 	libexpat-dev 
+RUN apt-get install -y 	python 
+RUN apt-get install -y 	python-serial 
+RUN apt-get install -y 	sed
+RUN apt-get install -y 	git 
+RUN apt-get install -y 	unzip 
+RUN apt-get install -y 	bash 	
+RUN apt-get install -y 	wget 
+RUN apt-get install -y 	bzip2 
+RUN apt-get install -y 	vim 
+RUN apt-get install -y 	screen 
+RUN apt-get install -y 	sudo
```

Comment:
Yes, please combined the install commands and most important, put the update command on the same line as otherwise it will cause caching issues, cf: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/apt-get